### PR TITLE
Is the linked list Plindrome?

### PR DESCRIPTION
**Pull request by Aarti002**
issue #679 solved

### CODE REVIEW

**Review by raksha009 — COMMENTED** [review]
Link your code with the file :)

**Comment by Aarti002** [comment]
i didn't get you, how to link code?

**Comment by raksha009** [comment]
> i didn't get you, how to link code?

Plz go through the guidelines and pinned issues.

File: Code/Python/isPalindrome_linked_list.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        ll=Node.insert_node(x)
+        n-=1
+    Node.viewll()
+    print('\n')
+    print(Node.isPalindrom(ll))
+
+#time complexity: O(n)
+#space : O(1)
+
+"""
```
Comment (krishna-NIT) [inline Code/Python/isPalindrome_linked_list.py:79]: Add Space and time complexity

Comment (krishna-NIT) [reply Code/Python/isPalindrome_linked_list.py:79]: ohh

File: Code/Python/isPalindrome_linked_list.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    Node = linked_list()
+    while n>0:
+        x = int(input())
+        ll=Node.insert_node(x)
+        n-=1
+    Node.viewll()
+    print('\n')
+    print(Node.isPalindrom(ll))
+
+#time complexity: O(n)
```
Comment (krishna-NIT) [inline Code/Python/isPalindrome_linked_list.py:76]: add what is  n there?

**Comment by krishna-NIT** [comment]
@Aarti002 Resolve the Conflict

**Comment by Aarti002** [comment]
@krishna-NIT I resolved it please check
Thank you

**Comment by krishna-NIT** [comment]
@rish-singhal LGTM

**Pull request merged**